package-lock.json

```diff
@@ -9933,7 +9933,7 @@
     },
     "eslint-config-prettier": {
       "version": "6.11.0",
-      "resolved": "https://registry.npmjs.org/eslint-config-prettier/-/eslint-config-prettier-6.11.0.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/eslint-config-prettier/-/eslint-config-prettier-6.11.0.tgz",
```

Comment:
This is using the morph registry. not sure this file should be change as there's no accompanying dependency updates

Reply:
Good spot.  I've annoyingly installed Morph and it keeps doing this.